Cargo.toml

```diff
@@ -194,3 +194,11 @@ os_pipe = ["io-lifetimes/os_pipe"]
 fs-err = ["io-lifetimes/fs-err"]
 #all-impls = ["async-std", "tokio", "os_pipe", "socket2", "mio", "fs-err"]
 all-impls = ["os_pipe", "fs-err"]
+
+# OS compatability features
+
+# Build for linux 4.11 or later
```

Comment:
Could you change this comment to say "Optimize for Linux 4.11 or later"? I think that would help clarify that it's still additive in terms of functionality, and it's just about tuning.